File: MODUL3/C/PRAK304_2410817310002_MuhammadRakhaAthallah.c
#include <stdio.h>

int main(){
    int a;
    printf("");
    scanf("%d", &a);

    if (a==0){ printf("Nol"); }
        else if (a>=1 && a<=9){ printf("Satuan"); }
        else if (a>=11 && a<=19){ printf("Belasan"); }
        else if (a>=10 && a<=99){ printf("puluhan"); }
        else {printf("Anda menginput Melebihi Limit Bilangan");  }
    }

File: MODUL3/PYTHON/PRAK304_2410817310002_MuhammadRakhaAthallah.py
a = int(input(""))

if a<0 or a>=100: print("Anda Menginput melebihi Limit Bilangan")
elif a==0: print("Nol")
elif 1<=a<=9: print("Satuan")
elif 11<=a<=19: print("belasan")
elif a==10 or a>=20 and a<=99: print("Puluhan")
else: print("Anda Menginput Melebihi Limit Bilangan")